apps/elasticsearch/templates/services/app/secret_certificates_passwords.yml.j2

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
```

Comment:
This secret should be created only if `elasticsearch_security_enabled` is set to true ?

Reply:
Ditto.